src/Testcontainers.WebDriver/WebDriverBuilder.cs

```diff
@@ -0,0 +1,178 @@
+using System.Text.Json;
+
+namespace Testcontainers.WebDriver;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class WebDriverBuilder : ContainerBuilder<WebDriverBuilder, WebDriverContainer, WebDriverConfiguration>
+{
+    public const string ChromeStandaloneImage = "selenium/standalone-chrome";
+    public const string FirefoxStandaloneImage = "selenium/standalone-firefox";
+    public const string EdgeStandaloneImage = "selenium/standalone-edge";
+    public const string OperaStandaloneImage = "selenium/standalone-opera";
+
+    public const ushort WebDriverPort = 4444;
+    public const ushort VncServerPort = 5900;
+    public const ushort NoVncServerPort = 7900;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="WebDriverBuilder" /> class.
+    /// </summary>
+    public WebDriverBuilder() : this(new WebDriverConfiguration())
+    {
+        DockerResourceConfiguration = Init().DockerResourceConfiguration;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="WebDriverBuilder" /> class.
+    /// </summary>
+    /// <param name="resourceConfiguration">The Docker resource configuration.</param>
+    private WebDriverBuilder(WebDriverConfiguration resourceConfiguration) : base(resourceConfiguration)
+    {
+        DockerResourceConfiguration = resourceConfiguration;
+    }
+
+    /// <inheritdoc />
+    protected override WebDriverConfiguration DockerResourceConfiguration { get; }
+
+    /// <summary>
+    /// Sets the WebDriver config.
+    /// </summary>
+    /// <param name="options">The options as string list for starting a hub or a node.</param>
+    /// <returns>A configured instance of <see cref="WebDriverBuilder" />.</returns>
+    public WebDriverBuilder WithConfigurationOptions(string options)
+    {
+        return Merge(DockerResourceConfiguration, new WebDriverConfiguration())
+            .WithEnvironment("SE_OPTS", options);
```

Comment:
I noticed you use often `Merge(DockerResourceConfiguration, new WebDriverConfiguration())`. If you do not set any property in the configuration object (`WebDriverConfiguration`) and just forward the member call to its base class you do not need to call `Merge(WebDriverConfiguration, WebDriverConfiguration)`. In the example above it is enough to return immediately.

```csharp
return WithEnvironment("SE_OPTS", options);
```

Reply:
o.k 
Thank you
I will change it